cmd/algod/main.go

```diff
@@ -344,7 +344,17 @@ func run() int {
 
 		// Send a heartbeat event every 10 minutes as a sign of life
 		go func() {
-			ticker := time.NewTicker(10 * time.Minute)
+			var interval time.Duration
+			defaultIntervalSecs := config.GetDefaultLocal().HeartbeatUpdateInterval
+			switch {
+			case cfg.HeartbeatUpdateInterval <= 0: // use default
+				interval = time.Second * time.Duration(defaultIntervalSecs)
+			case cfg.HeartbeatUpdateInterval < 60: // min frequency 1 minute
+				interval = time.Minute
```

Comment:
Could this be an error instead? Might be easier than having to document / explain why the units change depending on how big the number is.

Reply:
Alternatively, the units could be minutes, that way 1 is just the smallest non-disabled value.

Reply:
Ah, yes thats more reasonable. I read that as `time.Minute * interval`

Reply:
It could still be confusing. I'm imagining a local test where I set it to 5 to get some more information, then have to switch tasks to figure out why the events aren't being sent out.

Reply:
nit: I think changing units to minutes would be a slightly better option than changing the config at runtime